installation/scripts/utils.sh

```diff
@@ -96,15 +96,21 @@ function showFailedResources {
   fi
 }
 
+# checkInputParameterValue is a function to check if input parameter is valid
+# There HAS to be provided argument:
+# $1 - value for input parameter
+# for example in installation/cmd/run.sh we can set --vm-driver argument, which has to have a value.
+
 function checkInputParameterValue() {
-    if [ -z "${2}" ]; then
-        echo "Value parameter for ${1} is empty"
+    if [ -z "${1}" ]; then
+        echo "Value parameter is empty"
         echo "Make sure parameter value is neither empty nor start with two hyphens"
         exit 1
     fi
-    if [ "${2:0:2}" == "--" ]; then
-        echo "Invalid parameter value for ${1}:"
-        echo "${2}"
+
+    if [ "${1:0:2}" == "--" ]; then
```

Comment:
I value of parameter starts with -- it's not its value, it's another flag. In such case I wouldn't report "Invalid value", but "Missing value for --my-flag". In such case you can simplify this function to one `if`.